static-build/nemu/Dockerfile

```diff
@@ -0,0 +1,53 @@
+from ubuntu:16.04
+
+ARG NEMU_REPO
+# commit/tag/branch
+ARG NEMU_VERSION
+ARG NEMU_OVMF
+
+WORKDIR /root/nemu
+RUN apt-get update
```

Comment:
each `RUN` is a new layer in the final image, I wonder if won't be easier to use a script

Reply:
Agreed - I'd tend to try to keep the `Dockerfile` as small as possible and offload into a script.